app/scripts/views/form.js

```diff
@@ -176,6 +176,7 @@ function (_, $, p, Validate, AuthErrors, BaseView, Tooltip,
           }
 
           // all good, do the beforeSubmit, submit, and afterSubmit chain.
+          self.logScreenEvent('submit');
```

Comment:
@vladikoff - can you add this event to [client-metrics.md](https://github.com/mozilla/fxa-content-server/blob/master/docs/client-metrics.md#generic-events), probably under generic-events?